chat-staging.html

```diff
@@ -152,12 +161,18 @@
         changeLanguageWebChat(initialLanguage);
 
         // If caller is waiting for a counselor to connect, disable input (default language)
-        Twilio.FlexWebChat.MessageInput.defaultProps.disabledReason = translations[initialLanguage].MessageInputDisabledReasonHold;
         if (manager.chatClient) setChannelOnCreateWebChat(manager);
         
+        // Set caller name to be 'You'
+        Twilio.FlexWebChat.MessagingCanvas.defaultProps.memberDisplayOptions = {
+          yourDefaultName: 'You',
```

Comment:
@nickhurlburt Yes, it is being set here. We can localize it and/or use a name coming from a survey.

Reply:
Great!